docroot/themes/humsci/su_humsci_gin_admin/dist/su_humsci_gin_admin.css

```diff
@@ -134,3 +134,33 @@ html:not(.gin--dark-mode) .user-logged-in[data-gin-accent="custom"] {
 .menu-link-form .form-type--cshs .form-item__description {
   margin: 8px 0 0;
 }
+
+body.gin--edit-form .views-exposed-form .select-preact div > div {
+    min-width: 0 !important;
+    max-width: 240px;
+  }
+  
+body.gin--edit-form .views-exposed-form .select-preact div > div > div:first-of-type {
+  font-size: 0.875rem !important;
+  font-weight: 525 !important;
+  color: #222330;
```

Comment:
Gin has variables for most if its properties, please use them when available. In this case, `var(--gin-color-title)`.